docs/integrations/context-and-outputs.md

```diff
@@ -130,7 +130,12 @@ return_results(command_result)
 
 ---
 
-## Context Use Cases 
+## Context Use Cases
+
+:::caution Important Note
+When setting `integration_name` with the vendor value, make sure it matches the name of the initegration as defined in the [yml file](yaml-file#basic-information).
```

Comment:
```suggestion
When setting `integration_name` with the vendor value, make sure it matches the name of the integration as defined in the [yml file](yaml-file#basic-information).
```

Also i am not sure I understand this comment
I referred to ${DBotScore.Vendor}

Reply:
@aviadl yes, the integration_name will be set to `DBotScore.Vendor`.